forge-gui/res/cardsfolder/upcoming/nissa_resurgent_animist.txt

```diff
@@ -0,0 +1,10 @@
+Name:Nissa, Resurgent Animist
+ManaCost:2 G
+Types:Legendary Creature Elf Scout
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Land.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigMana | TriggerDescription$ Landfall — Whenever a land enters the battlefield under your control, add one mana of any color. Then if this is the second time this ability has resolved this turn, reveal cards from the top of your library until you reveal an Elf or Elemental card. Put that card into your hand and the rest on the bottom of your library in a random order.
+SVar:TrigMana:DB$ Mana | Produced$ Any | AILogic$ MostProminentInComputerHand | SubAbility$ RevealUntil
```

Comment:
This AILogic makes no sense at all

Reply:
I copied it from Lotus Cobra, you want it removed as well ?

Reply:
yes thanks!

Reply:
Actually wait, it does make sense
It just seems to be missing the implementation for it